web/src/message_list_view.js

```diff
@@ -1675,4 +1689,44 @@ export class MessageListView {
         });
         $rows_to_show_as_unread.addClass("unread");
     }
+
+    update_next_message_group_recipient_display_date($current_message_row, current_message) {
```

Comment:
Can you explain why we need this and comments here for it? Also, a screenshot of this in effect would be helpful.

Reply:
I think my main concern with this the performance cost of it since we will be calling it on every scroll.

Reply:
When we scroll up bottom message header may need to be changed if the year is not same. This effect is shown in the attached screenshot. Do we want to drop this change or add comments in the method execution section?

Reply:
done